third_party/terraform/tests/resource_container_node_pool_test.go.erb

```diff
@@ -202,6 +202,38 @@ func TestAccContainerNodePool_withWorkloadMetadataConfig(t *testing.T) {
 		},
 	})
 }
+
+func TestAccContainerNodePool_withWorkloadIdentityConfig(t *testing.T) {
+	t.Parallel()
+
+	pid := getTestProjectFromEnv()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckContainerClusterDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccContainerNodePool_withWorkloadMetadataConfig_gkeMetadataServer(pid),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("google_container_node_pool.with_workload_metadata_config",
+						"node_config.0.workload_metadata_config.0.node_metadata", "GKE_METADATA_SERVER"),
+				),
+			},
+			{
+				ResourceName:      "google_container_node_pool.with_workload_metadata_config",
+				ImportState:       true,
+				ImportStateVerify: true,
+				// Import always uses the v1 API, so beta features don't get imported.
```

Comment:
Was this your experience? Import should be using the beta API.

Reply:
Copy and paste carry over.